### PR TITLE
whitelist alerts pii

### PR DESCRIPTION
**Pull request by HemendraSharma04**

### CODE REVIEW

File: explorer/templates/explorer/play.html [hunk 1]
```diff
@@ -13,33 +13,38 @@
...
+            saved.</p>
         {% if lag_exists %}
-          <div class="alert alert-warning">Warning! The query results might be outdated since some data was last updated around {{ replication_lag }}.</div>
+        <div class="alert alert-warning">Warning! The query results might be outdated since some data was last updated
+            around {{ replication_lag }}.</div>
         {% endif %}
         </br>
         <p>
-            <b>Note:</b> Starting March 16th, access to the <b>phone_number</b> column has been removed for the following
+            <b>Note:</b> Starting March 16th, access to the <b>phone_number</b> column has been removed for the
```
Comment (Ujjwal-Squadstack) [inline explorer/templates/explorer/play.html]: Can you change its Date to April 15th, so we can identify easily whether we are running old explorer or new

File: explorer/templates/explorer/query_list.html [hunk 2]
```diff
@@ -2,123 +2,125 @@
...
 {% block sql_explorer_navlinks %}
-    {% if can_change %}
-      <li><a href="{% url "query_create" %}">New Query</a></li>
-      <li><a href="{% url "explorer_playground" %}">Playground</a></li>
-      <li><a href="{% url "explorer_logs" %}">Logs</a></li>
-      <li><a href="{% url "explorer_change_logs" %}">Change Logs</a></li>
-    {% endif %}
+{% if can_change %}
+<li><a href="{% url "query_create" %}">New Query</a></li>
+<li><a href="{% url "explorer_playground" %}">ground</a></li>
```
Comment (Ujjwal-Squadstack) [inline explorer/templates/explorer/query_list.html]: Why we changed this to ground?

File: explorer/templates/explorer/query_list.html [hunk 3]
```diff
@@ -2,123 +2,125 @@
...
                 <th>Created</th>
                 {% if tasks_enabled %}
-                  <th>Email</th>
+                <th>Email</th>
                 {% endif %}
                 <th>CSV</th>
                 {% if can_change %}
-                    <th>Play</th>
-                    <th>Delete</th>
+                <th>Playground</th>
```
Comment (Ujjwal-Squadstack) [inline explorer/templates/explorer/query_list.html]: Why we renamed this as Playground?

**Review by Ujjwal-Squadstack — CHANGES_REQUESTED** [review]

File: explorer/models.py [hunk 4]
```diff
@@ -47,8 +50,9 @@ def passes_blacklist(self):
     def final_sql(self):
         return swap_params(self.sql, self.available_params())
 
-    def execute_query_only(self):
-        return QueryResult(self.final_sql())
+    def execute_query_only(self, connection_type=None):
```
Comment (Ujjwal-Squadstack) [inline explorer/models.py]: Can you change this param name to something suitable?

**Review by Ujjwal-Squadstack — CHANGES_REQUESTED** [review]

File: explorer/templates/explorer/query_list.html (hunk 2 above)
Comment (HemendraSharma04) [reply explorer/templates/explorer/query_list.html]: for testing i'll change it 👍🏻

**Review by Ujjwal-Squadstack — APPROVED** [review]

**Pull request merged**